src/stratigraphy/main.py

```diff
@@ -276,6 +277,8 @@ def transform_groups(
 def merge_blocks_by_vertical_spacing(blocks: list[TextBlock], target_merge_count: int) -> list[TextBlock]:
     """Merge textblocks without any geometric lines that separates them.
 
+    Note: Deprecated. Currently not in use any more. Kept here until we are sure that it is not needed anymore.
```

Comment:
I'd be happy to completely remove this method already now. I don't expect that we will reinstate it in the future (and if we still want to, then we can always use the git history).